good_job.gemspec

```diff
@@ -54,6 +54,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "fugit", ">= 1.1"
   spec.add_dependency "railties", ">= 5.2.0"
   spec.add_dependency "thor", ">= 0.14.1"
+  spec.add_dependency "webrick", ">= 1.3"
```

Comment:
I didn't put much thought into it, though I'm not quite sure how I came to `v1.3` as it looks like that shipped with Ruby 2.4, which GoodJob does not support. 

I think of the gemspec as descriptive of compatibility; I think security should be considered at the application that is consuming the gemspec. Trying to balance:

- if the application is not previously including webrick, that bundler is not defaulting to an insecure version (I don't know how Bundler decides whether to pull the latest version, or just whatever it has lying around locally)
- if the application is previously including webrick, not forcing the application to update the version of webrick because that might be a blocker for the adopting/updating GoodJob in a controlled environment.

The other thing I did consider, rather than pulling in webrick, was just pasting in one of the many "rack-compatible http server in 30 lines of ruby" and packaging it directly. These health check requests are very, very simple and really only intended to be used internally.